manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -199,6 +222,14 @@ global:
     #   cpu: 100m
     #   memory: 128Mi
 
+  # Default node tolerations to be applied to all deployments so that all pods can be
+  # scheduled to a particular nodes with matching taints. Each component can overwrite
+  # these default values by adding its tolerations block in the relevant section below
+  # and setting the desired values.
+  # Configure this field in case that all pods of Istio control plane are expected to
+  # be scheduled to particular nodes with specified taints.
+  defaultTolerations: []
+
```

Comment:
I think you'll also need to add `global.arch`?

Reply:
Thanks @dgn